src/com/projectkorra/projectkorra/configuration/ConfigManager.java

```diff
@@ -512,12 +518,19 @@ public static void configCheck(final ConfigType type) {
 			plantBlocks.add(Material.SUNFLOWER.toString());
 			plantBlocks.add(Material.POPPY.toString());
 			plantBlocks.add(Material.FERN.toString());
+			plantBlocks.add(Material.LILY_OF_THE_VALLEY.toString());

```

Comment:
![image](https://user-images.githubusercontent.com/9260792/87116743-6dd0bc80-c22b-11ea-9c91-035fae6f69a0.png)
Am I a joke to you?

{You forgot about the cornflower)

Reply:
LOL